week1/merge.py

```diff
@@ -0,0 +1,49 @@
+"""
+Merge function for 2048 game.
+"""
+
+def merge(line):
+    """
+    Function that merges a single row or column in 2048.
+    """
+    line_without_zeros = [value for value in line if value != 0]
+    merged_line = []
+    
+    try_merge(line_without_zeros, merged_line)
+    while len(line) - len(merged_line) > 0:
```

Comment:
I think line 10 and this loop can be simplified. How? :-)

Reply:
I ended up doing something like this... 
```
[0 for dummy_num in range(len(line) - len(line_without_zeros))]
```
Is that close to what you were thinking?
Also, the class I am taking suggests naming unused variables in python for loops dummy_. Is there a better way to handle this situation?